src/main/java/org/pentaho/mongo/wrapper/collection/MongoCollectionWrapper.java

```diff
@@ -37,7 +37,9 @@
 
   MongoCursorWrapper find( DBObject dbObject, DBObject dbObject2 ) throws MongoDbException;
 
-  AggregationOutput aggregate( DBObject firstP, DBObject[] remainder ) throws MongoDbException;
+  Cursor aggregate( List<? extends DBObject> pipeline, AggregationOptions options );
```

Comment:
Are we sure we can safely remove methods from API? Wouldn't it be more sensible to deprecate the old method?